lua/tabnine/status.lua

```diff
@@ -17,7 +17,9 @@ local function poll_service_level()
 		5000,
 		vim.schedule_wrap(function()
 			tabnine_binary:request({ State = { dummy_property = true } }, function(response)
-				if response.service_level == "Pro" or response.service_level == "Trial" then
+				if not response then
+					service_level = "unknown" -- Avoid 'loading' forever.
```

Comment:
I did this mostly to help us in future issues. If we get a report where it says `loading` forever, then it could be almost anything causing it, while if it says `unknown`, we now immediately know the cause was a NULL. I can remove it if you'd like, but right now i suspect we are covering up a bug in the binary and I'm hoping we get a report in the future with a smoking gun

Reply:
Let's keep the old logic for time being

Reply:
I reverted this in [06ee6d1](https://github.com/codota/tabnine-nvim/pull/129/commits/06ee6d154e5b1ef6dbbf0526f2c904c29042ca4b)